zh/enable-tls-between-components.md

```diff
@@ -427,6 +427,80 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tls-between-components/']
         cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=internal tiflash-server.json | cfssljson -bare tiflash-server
         ```
 
+    - TiKV-Importer Server 端证书      
+
+        如需要[使用 TiDB Lightning 恢复 Kubernetes 上的集群数据](restore-data-using-tidb-lightning.md)，则需要为其中的 TiKV-Importer 组件生成如下的 Server 端证书。
+
+        首先生成默认的 `importer-server.json` 文件：
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        cfssl print-defaults csr > importer-server.json
+        ```
+
+        然后编辑这个文件，修改 `CN`、`hosts` 属性：
+
+        ```json
+        ...
+            "CN": "TiDB",
+            "hosts": [
+              "127.0.0.1",
+              "::1",
+              "${cluster_name}-importer",
+              "${cluster_name}-importer.${namespace}",
+              "${cluster_name}-importer.${namespace}.svc"
```

Comment:
Should we also add the below items?
```
              "*.${cluster_name}-importer",
              "*.${cluster_name}-importer.${namespace}",
              "*.${cluster_name}-importer.${namespace}.svc"
```
Does the Grafana dashboard show the metrics of the importer correctly?

Reply:
Does it seem we also not added these items for other components? like

```
...
    "CN": "TiDB",
    "hosts": [
      "127.0.0.1",
      "::1",
      "${cluster_name}-tidb",
      "${cluster_name}-tidb.${namespace}",
      "${cluster_name}-tidb.${namespace}.svc",
      "${cluster_name}-tidb-peer",
      "${cluster_name}-tidb-peer.${namespace}",
      "${cluster_name}-tidb-peer.${namespace}.svc",
      "*.${cluster_name}-tidb-peer",
      "*.${cluster_name}-tidb-peer.${namespace}",
      "*.${cluster_name}-tidb-peer.${namespace}.svc"
    ],
...
```

Reply:
Yes, sorry, I misunderstand that.
BTW, the monitoring works as expected for both lightning and importer, right?

Reply:
I tested manually for importer, both Prometheus and Grafana can query metrics.

but for Lightning, I only have a small dataset, and Lightning completed too fast before metrics have been pulled by Prometheus.

Reply:
OK.